eslint/local-eslint-config/config/heft.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/heft/v0/heft.schema.json",
+
+  "phasesByName": {
+    "build": {
+      "cleanFiles": [{ "includeGlobs": ["mixins", "patch", "profile"] }],
```

Comment:
What if any of these takes a dependency on an eslint plugin that this project doesn't bring along?

Reply:
Build of downstream projects will fail. It'll be pretty obvious.